### PR TITLE
[2565] - Add Dockerfile for background geolocating process

### PR DESCRIPTION
**Pull request by TobyRet**
### Context
This will be run as a separate process. Any updates to Site or Provider addresses will trigger the geolocation job.

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by TobyRet** [comment]
[Add geocoding setup](https://trello.com/c/hDXI6Vse/2565-add-geocoding-setup)

**Comment by TobyRet** [comment]
I've now added `bg-find-sync/` `bg-geocode/` to CodeClimate exclude patterns ...

**Comment by TobyRet** [comment]
[Create Azure resource for running background geolocation process](https://trello.com/c/YEbCtAFJ/2635-create-azure-resource-for-running-background-geolocation-process)

File: bg-geocode/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ARG  DOCKER_REPOSITORY
+ARG  MANAGE_COURSES_BACKEND_IMAGE
```
Comment (misaka) [inline bg-geocode/Dockerfile]: @rizzkhan1 also [includes this file in his PR](https://github.com/DFE-Digital/manage-courses-backend/pull/1032/files#diff-ce4e39c0db023b61c0fa12db36c3d60c). My comment there was just that the naming sounds like this is the `manage-courses-backend`. Technically, of course, it is ... but I just wonder if it wouldn't be confusing one day. Just a minor issue, though.

**Comment by TobyRet** [comment]
Closing as @rizzkhan1 has already done this work
https://github.com/DFE-Digital/manage-courses-backend/pull/1032/files#diff-ce4e39c0db023b61c0fa12db36c3d60c

**Pull request closed without merging**